src/ui/components/LedgerAdmin.js

```diff
@@ -137,12 +166,12 @@ export const LedgerAdmin = (): ReactNode => {
         />
         {selectedId && (
           <FormControlLabel
-            fullWidth
+            fullwidth="true"
             className={classes.checkboxElement}
             control={
               <Checkbox
-                checked={checkboxSelected}
-                onChange={() => toggleIdentityActivation(selectedId)}
+                checked={NullUtil.orElse(isChecked[selectedId], false)}
+                disabled={true}
```

Comment:
this would disable the checkbox permanently, seems like a typo or left over debugging stuff?